flake8_print.py

```diff
@@ -1,54 +1,38 @@
-import ast
-import tokenize
+"""Extension for flake8 that finds usage of print."""
+import re
 
-from sys import stdin
-
-__version__ = '1.6.1'
+__version__ = '1.7.0'
 
 PRINT_ERROR_CODE = 'T001'
 PRINT_ERROR_MESSAGE = 'print statement found.'
 
 
-class PrintStatementChecker(object):
-    name = 'flake8-print'
-    version = __version__
-
-    def __init__(self, tree, filename='(none)', builtins=None):
-        self.tree = tree
-        self.filename = (filename == 'stdin' and stdin) or filename
-
-    def run(self):
-        if self.filename == stdin:
-            noqa = get_noqa_lines(self.filename)
-        else:
-            with open(self.filename, 'r') as file_to_check:
-                noqa = get_noqa_lines(file_to_check.readlines())
-
-        errors = check_tree_for_print_statements(self.tree, noqa)
-
-        for error in errors:
-            yield (error.get("line"), error.get("col"), error.get("message"), type(self))
-
+def flake8ext(f):
+    """Decorate flake8 extension function."""
+    f.name = 'flake8-print'
+    f.version = __version__
+    return f
 
-def get_noqa_lines(code):
-    tokens = tokenize.generate_tokens(lambda L=iter(code): next(L))
-    noqa = [token[2][0] for token in tokens if token[0] == tokenize.COMMENT and (token[1].endswith('noqa') or (isinstance(token[0], str) and token[0].endswith('noqa')))]
-    return noqa
 
+RE_PRINT_STATEMENT = re.compile(r"(?<![=\s])\s*print\s+[^(=]", re.MULTILINE)
```

Comment:
the `\s` in `[=\s]` is ensuring that whitespace is matched against `\s*`.

otherwise '= print' will result in `\s*` being equal to nothing, and the space character matching `(?<![=])` , as space is indeed not the equal sign.